karspexet/ticket/models.py

```diff
@@ -90,3 +90,6 @@ class PricingModel(models.Model):
 
     def price_for(self, ticket_type):
         return int(self.prices[ticket_type])
+
+    def __repr__(self):
+        return (self.prices, self.seating_group, self.valid_from)
```

Comment:
Shouldn't `repr` return a string? You can use formatting strings like so `f"text{python expression}"`, add `{...!r}` to make it interpolate its `repr` string.

Reply:
Yes, in 3.6 we can. The `.tool-versions` file still thinks we use 3.5 though, which isn't even true. We use 3.6 on both CI and on the server, so I'll update `.tool-versions` as well.